edison/static/js/login-signup.js

```diff
@@ -0,0 +1,81 @@
+var right_edge
+
+function ajaxCall(url, data, method, successCallback, errorCallback) {
+	$.ajax({
+		method: method,
+		dataType: "json",
+		url: url,
+		data: data,
+		success: successCallback,
+		error: errorCallback
+	});
+}
+
+function signupBtn() {
+	ajaxCall(
+		'signup',
+		{
+			'username': $("#main_form_email").val(),
+			'password': $("#main_form_password").val()
+		}, "POST", 
+		function(data) {
+			window.location.replace("/");
```

Comment:
Using replace will replace the current document and replace the current history with that URL, making it so you can't go back to the previous document loaded.
Is this behavior is what you want ?
If not change it to assign('/').

Reply:
ack